### PR TITLE
Accept luke-cage-preview to enable required_approving_review_count

### PR DESCRIPTION
**Pull request by fejta**
/assign @sebastienvas @cjwagner 

See https://developer.github.com/v3/repos/branches/#update-branch-protection

### CODE REVIEW

File: prow/github/client.go [hunk 1]
```diff
@@ -857,6 +857,7 @@ func (c *Client) RemoveBranchProtection(org, repo, branch string) error {
 func (c *Client) UpdateBranchProtection(org, repo, branch string, config BranchProtectionRequest) error {
 	c.log("UpdateBranchProtection", org, repo, branch, config)
 	_, err := c.request(&request{
+		accept:      "application/vnd.github.luke-cage-preview+json", // for required_approving_review_count
```
[the remark this answers is not in the document]
Comment (fejta) [reply prow/github/client.go]: apparently we are not the only ones with a sense of humor 😁

[the remark this answers is not in the document]
Comment (BenTheElder) [reply prow/github/client.go]: or long/mixed-delimiter-names+json_blobs 🙃

**Comment by fejta** [comment]
Will cut a new prow release tomorrow.

**Pull request merged**